Lib/test/test_bz2.py

```diff
@@ -106,6 +106,8 @@ def testBadArgs(self):
         # compresslevel is keyword-only
         self.assertRaises(TypeError, BZ2File, os.devnull, "r", 3)
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
```

Comment:
Why should we go this way if it occures these many regressions?
If this can be fixed in future, let's fix it first and merge it later.
Otherwise this must be an optional feature rather for specific requirements.